src/test/java/com/atlassian/db/replica/api/AuroraMultiReplicaConsistencyTest.java

```diff
@@ -0,0 +1,103 @@
+package com.atlassian.db.replica.api;
+
+import com.atlassian.db.replica.internal.aurora.AuroraReplicaNode;
+import com.atlassian.db.replica.spi.ReplicaConsistency;
+import com.atlassian.db.replica.spi.SuppliedCache;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class AuroraMultiReplicaConsistencyTest {
+    @Mock
+    private Connection supplierConnection;
+    @Mock
+    private ReplicaConsistency replicaConsistency;
+    @Mock
+    private SuppliedCache<Collection<Database>> discoveredReplicasCache;
+
+    private AuroraMultiReplicaConsistency sut;
+
+    @BeforeEach
+    void beforeEach() throws SQLException {
+        sut = AuroraMultiReplicaConsistency.builder()
+            .replicaConsistency(replicaConsistency)
+            .discoveredReplicasCache(discoveredReplicasCache)
+            .build();
+    }
+
+    private Collection<Connection> mockReplicaConnections(int count) {
+        Collection<Connection> replicas = new LinkedList<>();
+        Collection<Database> nodes = new LinkedList<>();
+
+        for (int i = 0; i < count; i++) {
+            Connection replica = mock(Connection.class);
+            replicas.add(replica);
+
+            AuroraReplicaNode node = new AuroraReplicaNode(
+                String.valueOf(i),
+                () -> replica
+            );
+            nodes.add(node);
+        }
+        when(discoveredReplicasCache.get(any())).thenReturn(Optional.of(nodes));
+
+        return replicas;
+    }
+
+    @Test
+    void shouldNotBeConsistentWhenNoReplicasAreAvailable() throws SQLException {
```

Comment:
So when all replicas are down, and only the main database survived, I will never move forward.

Reply:
We should change it, no inconsistent replicas means we are consistent.

Reply:
will change it so that no replicas indicate that we are consistent